### PR TITLE
Add instructions about which dependencies we should update on build

### PR DESCRIPTION
**Pull request by dmitrykuzmenkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +58,35 @@ xattr -dr com.apple.quarantine manticore-executor
...
+* build-linux
+* build-osx
+
+Scripts accept a version of PHP as a parameter. The current version is `8.1.11`. To build the binary, you should run the following example:
+
+```bash
+./build-linux "8.1.11"
+```
+
+The command above will build the package on Linux with **PHP** `8.1.11`. Once it's done, you can find your binary in the folder `dist/bin`.
```
Comment (sanikolaev) [inline README.md:92]:
```suggestion
The command above will build the package on Linux with **PHP** `8.1.11`. Once it's done, you can find your binary in folder `dist/bin`.
```

File: README.md [hunk 2]
```diff
@@ -57,3 +58,35 @@ xattr -dr com.apple.quarantine manticore-executor
...
+   * Centos 7/8 x86_64 and arm64
+   * Ubuntu Xenial/Bionic/Focal/Jammy x86_64 and arm64
+   * Debian Stretch/Buster/Bullseye x86_64 and arm64
+   * MacOS x86_64
+
+2. The same workflow deploys the packages to [repositories](https://repo.manticoresearch.com)
+
+3. What's to be done manually after the above is:
+
+   * build binaries for MacOS arm64 following [this instruction](#how-to-build-manually)
```
Comment (sanikolaev) [inline README.md:75]: wrong link?

File: README.md [hunk 3]
```diff
@@ -57,3 +58,35 @@ xattr -dr com.apple.quarantine manticore-executor
...
+   * update `root_url` [Homebrew formula](https://github.com/manticoresoftware/homebrew-manticore/blob/main/Formula/manticore-executor.rb) file with latest release version
```
Comment (sanikolaev) [inline README.md:76]:
```suggestion
   * update `root_url` in the [Homebrew formula](https://github.com/manticoresoftware/homebrew-manticore/blob/main/Formula/manticore-executor.rb) file with the latest release version
```

File: README.md [hunk 4]
```diff
@@ -57,3 +58,35 @@ xattr -dr com.apple.quarantine manticore-executor
...
+   * update `DEB_PKG` variables in the [Dockerfile](https://github.com/manticoresoftware/manticoresearch-backup/blob/main/Dockerfile) of [manticoresearch-backup](https://github.com/manticoresoftware/manticoresearch-backup) repository
+
+## How to build manually
+
+Several scripts used to build the final package:
+
+* build-linux
+* build-osx
```
Comment (sanikolaev) [inline README.md:84]:
```suggestion
* `build-linux`
* `build-osx`
```

File: README.md [hunk 5]
```diff
@@ -57,3 +58,35 @@ xattr -dr com.apple.quarantine manticore-executor
...
+
+Scripts accept a version of PHP as a parameter. The current version is `8.1.11`. To build the binary, you should run the following example:
```
Comment (sanikolaev) [inline README.md:86]:
```suggestion
The scripts accept a version of PHP as a parameter. The current version is `8.1.11`. To build the binary, you should run the following example:
```

**Review by sanikolaev — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dmitrykuzmenkov) [reply README.md:75]: The link points to self documentation – it's correct

**Review by sanikolaev — APPROVED** [review]

**Pull request merged**